musician-details.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import Modal from 'react-bootstrap4-modal'
+
+export default class MusicianDetails extends React.Component {
```

Comment:
If your component only has a `render` method, it's better to use a [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components).